src/libs/Violations/ViolationsUtils.ts

```diff
@@ -7,7 +7,14 @@ import * as TransactionUtils from '@libs/TransactionUtils';
 import CONST from '@src/CONST';
 import type {TranslationPaths} from '@src/languages/types';
 import ONYXKEYS from '@src/ONYXKEYS';
-import type {PolicyCategories, PolicyTagList, Transaction, TransactionViolation} from '@src/types/onyx';
+import type {PolicyCategories, PolicyTagList, Transaction, TransactionViolation, ViolationName} from '@src/types/onyx';
+
+const TRANSACTION_VIOLATIONS_FILTER = [
```

Comment:
Please move this to a local variable right above where it's used. I don't think this needs to be a const outside of the scope of that function.

Let's call it `tagViolations`